Cython/Compiler/ExprNodes.py

```diff
@@ -7690,7 +7690,7 @@ def calculate_access_code(self):
                     # sure to get the right fused name, as our entry was
                     # replaced by our parent index node
                     # (AnalyseExpressionsTransform)
-                    self.member = self.entry.cname
+                    self.member = '.'.join(self.member.split('.')[0:-1] + [self.entry.cname])
```

Comment:
Here, I had to fix the case when `self.member` contains `__pyx_base` prefix but `self.entry.cname` does not. I suspect this needs to be fixed elsewhere but not sure where currently...

Reply:
I tend to agree that this is in the wrong place.I suspect it should be somewhere around `PyrexTypes.get_fused_cname`/`PyrexTypes.specialize_entry` but it isn't hugely obvious how to do that.

Doing it there at the point that the entry is created would be largely consistent with regular non-fused `cdef` functions.